pygeos/creation.py

```diff
@@ -125,45 +150,94 @@ def linearrings(coords, y=None, z=None, indices=None, **kwargs):
 
 
 @multithreading_enabled
-def polygons(shells, holes=None, indices=None, **kwargs):
+def polygons(geometries, holes=None, indices=None, **kwargs):
     """Create an array of polygons.
-
-    Parameters
-    ----------
-    shell : array_like
-        An array of linearrings that constitute the out shell of the polygons.
-        Coordinates can also be passed, see linearrings.
-    holes : array_like or None
-        An array of lists of linearrings that constitute holes for each shell.
-    indices : array_like or None
-        Indices into the shells array where input holes belong. If
-        provided, shells, holes, and indices should all be 1D and the size
-        of holes must equal the size of indices.
-    **kwargs
-        For other keyword-only arguments, see the
-        `NumPy ufunc docs <https://numpy.org/doc/stable/reference/ufuncs.html#ufuncs-kwargs>`_.
-        Ignored if ``indices`` is provided.
+    and the holes; the first geometry for each index is the outer shell and all subsequent geometries in that index are the holes...
```

Comment:
This seems a small copy/paste or merge error. I think the line can just be removed? (it's content from the `indices` keyword explanation)